modules/ROOT/pages/configure-global-as2-settings.adoc

```diff
@@ -0,0 +1,17 @@
+= Configuring Global AS2 Endpoint Settings
+
+You can prevent Anypoint Partner Manager from processing requests with expired certificates by following these steps:
+
+. Access the <host> page.
+. To prevent inbound requests with expired certificates from being processed by Partner Manager:
```

Comment:
For this and line 10, suggest making these bullets since they're not sequential (unless they are).
. To prevent:
** Inbound requests...
** Outbound requests...